install.sh

```diff
@@ -127,26 +154,24 @@ exists_but_not_writable() {
 }
 
 get_owner() {
-  stat -f "%u" "$1"
+  $STAT "%u" "$1"
 }
 
 file_not_owned() {
   [[ "$(get_owner "$1")" != "$(id -u)" ]]
 }
 
 get_group() {
-  stat -f "%g" "$1"
+  $STAT "%g" "$1"
 }
 
 file_not_grpowned() {
   [[ " $(id -G "$USER") " != *" $(get_group "$1") "*  ]]
 }
 
-# USER isn't always set so provide a fall back for the installer and subprocesses.
-if [[ -z "$USER" ]]; then
-  USER="$(chomp "$(id -un)")"
-  export USER
-fi
+# USER isn't always set so always get it from `id` for the installer and subprocesses.
+USER="$(chomp "$(id -un)")"
```

Comment:
Why this change if `USER` is already set?

Reply:
Because if it isn't set, the script will fail when checking if it exists, because of `set -u` line at the top of the script.

Reply:
Could put this check above that `set`; want to respect the existing variable if it's there.

Reply:
Or you can use `if [[ -z "${USER-}" ]]`.

Reply:
That's neat, thanks @itchyny!